master/src/test/java/org/evosuite/ga/metaheuristics/CellularGASystemTest.java

```diff
@@ -0,0 +1,73 @@
+package org.evosuite.ga.metaheuristics;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.evosuite.EvoSuite;
+import org.evosuite.Properties;
+import org.evosuite.SystemTestBase;
+import org.evosuite.Properties.Algorithm;
+import org.evosuite.Properties.Criterion;
+import org.evosuite.Properties.StoppingCondition;
+import org.evosuite.ga.Chromosome;
+import org.evosuite.utils.Randomness;
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.examples.with.different.packagename.ClassHierarchyIncludingInterfaces;
+import com.examples.with.different.packagename.XMLElement2;
+
+/**
+ * CellularGA system test
+ * @author Nasser Albunian
+ *
+ */
+public class CellularGASystemTest extends SystemTestBase{
+	
+	public List<Chromosome> setup(StoppingCondition sc, int budget, String cut){
+		Properties.CRITERION = new Criterion[1];
+		Properties.CRITERION[0] = Criterion.BRANCH;
+		Properties.ALGORITHM = Algorithm.CELLULARGA;
+	    Properties.POPULATION = 50;
+	    Properties.STOPPING_CONDITION = sc;
+	    Properties.SEARCH_BUDGET = budget;
+	    Properties.MINIMIZE = false;
+	    Randomness.setSeed(10);
```

Comment:
You don't need to set the _seed_, SystemTestBase class already does that for you.